### PR TITLE
Adds example to review request body for troubleshooting.

### LINKED ISSUE #13: Demonstrate how to review the request body for troubleshooting
When debugging, it can be useful to examine the request body before sending to the SendGrid API.

Prove an example of how to examine the request body in TROUBLESHOOTING.md. For an example, see [here](https://github.com/sendgrid/sendgrid-python/blob/master/TROUBLESHOOTING.md#request-body).

### PR DESCRIPTION
**Pull request by mukulmishra18**
Fixes #13 .

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/nodejs-http-client?pullRequest=30) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/nodejs-http-client?pullRequest=30) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/nodejs-http-client?pullRequest=30) it.</sub>

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* [Viewing the Request Body](#request-body)
+
+<a name="request-body"></a>
+## Viewing the Request Body
+
+When debugging or testing, it may be useful to examine the raw request body to compare against the [documented format](https://sendgrid.com/docs/API_Reference/api_v3.html).
+
+You can do this with something like:
+```javascript
+var requestPost = client.emptyRequest()
```
[the remark this answers is not in the document]
Comment (mukulmishra18) [reply TROUBLESHOOTING.md]: Sorry, I did not get it properly. I think I already added how to log request body [here](https://github.com/mukulmishra18/nodejs-http-client/commit/a52e67728426aac224ca7ce9819e15fc33d37552#diff-e8f0e79b9005fb9fde8c970e2d1b6281R23), before sending the POST request. Should I have to log `request.body` here also?

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+If you have a non-library SendGrid issue, please contact our [support team](https://support.sendgrid.com).
+
+If you can't find a solution below, please open an [issue](https://github.com/sendgrid/sendgrid-python/issues).
```
[the remark this answers is not in the document]
Comment (mukulmishra18) [reply TROUBLESHOOTING.md]: Sorry, my bad! I will fix it. Thanks.

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+requestPost.method = 'POST'
+requestPost.path = '/v3/api_keys'
+requestPost.body = { data: 'some test data' }
+requestPost.headers['X-Test'] = 'test'
+
+// Log request body before sending POST request.
+console.log(requestPost.body)
+
+client.API(requestPost, (response) => { })
+```
```
[the remark this answers is not in the document]
Comment (mukulmishra18) [reply TROUBLESHOOTING.md]: Done with new commit. Thanks.

**Comment by mukulmishra18** [comment]
@mbernier can you please check this PR and let me know if anything else needs to be done?

**Comment by mbernier** [comment]
I think we're good to go!

**Comment by thinkingserious** [comment]
Hello @mukulmishra18,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by mukulmishra18** [comment]
Hi @thinkingserious, Filled the form. Thanks!

**Pull request merged**